.github/workflows/gh-pages.yml

```
@@ -216,28 +216,18 @@ jobs:
-          echo "$HOME/.local/bin" >> $GITHUB_PATH
-
-      - name: Install Dependencies
-        if: steps.cache-woff2.outputs.cache-hit != 'true'
-        working-directory: fonts
-        run: |
-          set -e
-          poetry install
-
       - name: Convert and Copy Woff2
         if: steps.cache-woff2.outputs.cache-hit != 'true'
         working-directory: fonts
```

actionlint (1.7.7)

"working-directory" is not available with "uses". it is only available with "run"

(syntax-check)